src/cascadia/TerminalApp/AppActionHandlers.cpp

```diff
@@ -1348,6 +1348,16 @@ namespace winrt::TerminalApp::implementation
                     }
                 }
 
+                if (WI_IsFlagSet(source, SuggestionsSource::WinGetCommandNotFound) &&
+                    context != nullptr)
+                {
+                    const auto recentCommands = Command::ToSendInputCommands(context.WinGetSuggestions());
+                    for (const auto& t : recentCommands)
+                    {
+                        commandsCollection.push_back(t);
+                    }
+                }
+

```

Comment:
@zadjii-msft is this kinda what you meant by adding the new suggestions to the context? If that's the case, we should rename the context to something else. Maybe `ShellIntegrationContext`?

Reply:
**Exactly** what I was thinking. My quick recommendations:
* I'd probably not name the member `WinGetSuggestions`, but instead `QuickFixes`. I'd think these should come through just the same as any other "quick fix" we add to the control (i.e. like a VT-driven one). 
* the same thing but with the flag in `SuggestionsSource` - that pobably also needs to get added to the `JSON_FLAG_MAPPER` in `TerminalSettingsSerializationHelpers.h`, as `quickFixes`
* we probably want to pass a `\uEA80` icon in here.

Reply:
oofda I see the comment about the OEM icon below now. Well, similar idea I guess.

Reply:
Oh huh, maybe we do want to give the winget suggestions a different icon. That's actually a really fun idea.